### PR TITLE
EG-1092, Number Lists

### PR DESCRIPTION
**Pull request by cmrhmrc**

### CODE REVIEW

File: app/views/components/numbered_circle_list.scala.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.ui._
+@import controllers.routes._
```
Comment (LesWilson) [inline app/views/components/numbered_circle_list.scala.html:18]: Minor - import not required

File: test/services/AggregatorSpec.scala [hunk 2]
```diff
@@ -63,7 +63,186 @@ class RowAggregatorSpec extends BaseSpec {
...
+
+    aggregatedStanzas(1) shouldBe NumListGroup(Seq(num1ListCo))
+  }
+
+  "create two number list groups for two contiguous rows with stack set to false" in new NumberedListTest {
+
+    val stanzas: Seq[VisualStanza] = Seq(
+      callout,
+      num1ListCo,
+      num1ListCo,
```
Comment (LesWilson) [inline test/services/AggregatorSpec.scala:112]: Should this be num2ListCo?

Comment (LesWilson) [reply test/services/AggregatorSpec.scala:112]: actually test later with num2ListCo so this test just reuses num1ListCo as only one with stacked false - so ignore comment above :)

File: test/services/UIBuilderSpec.scala [hunk 3]
```diff
@@ -354,7 +354,87 @@ class UIBuilderSpec extends BaseSpec with ProcessJson {
     val headingText = Text(headingPhrase.langs)
   }
 
+  trait NumberListTest extends Test {
+    val headingPhrase = Phrase(Vector("Heading", "Heading"))
+    val num1Phrase = Phrase(Vector("Line1", "Welsh Line1"))
+    val num2Phrase = Phrase(Vector("Line1", "Welsh Line1"))
+    val num3Phrase = Phrase(Vector("Line1", "Welsh Line1"))
+    val num4Phrase = Phrase(Vector("Line1", "Welsh Line1"))
```
Comment (LesWilson) [inline test/services/UIBuilderSpec.scala:362]: Doesn't look like these phrase values get used anywhere - just set on the callouts - all say Line1 - could use a single phrase if no checking done in tests of the phrase order

**Review by LesWilson — CHANGES_REQUESTED** [review]
Some very minor unused imports and a point on test content

File: app/views/components/numbered_list.scala.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.ui._
+@import controllers.routes._
```
[the remark this answers is not in the document]
Comment (cmrhmrc) [reply app/views/components/numbered_list.scala.html:18]: done

File: app/views/components/numbered_circle_list.scala.html (hunk 1 above)
Comment (cmrhmrc) [reply app/views/components/numbered_circle_list.scala.html:18]: done

File: test/services/AggregatorSpec.scala (hunk 2 above)
Comment (cmrhmrc) [reply test/services/AggregatorSpec.scala:112]: ok

File: test/services/UIBuilderSpec.scala (hunk 3 above)
Comment (cmrhmrc) [reply test/services/UIBuilderSpec.scala:362]: updated

**Review by LesWilson — APPROVED** [review]

**Pull request merged**